src/ui/settings/ViewSettingsMenu.qml

```diff
@@ -105,7 +105,19 @@ VPNFlickable {
                               manageAccountButton.y + manageAccountButton.height)
         width: parent.width - Theme.windowMargin
         anchors.horizontalCenter: parent.horizontalCenter
+        VPNSettingsItem {
+            objectName: "settingsWhatsNew"
+
+            settingTitle: "What’s new"
+            imageLeftSrc: "../resources/gift-dark.svg"
+            imageRightSrc: "../resources/chevron.svg"
+            onClicked: settingsStackView.push("../settings/ViewWhatsNew.qml")
+            showIndicator: !VPNSettings.newFeaturesSeen
```

Comment:
What about new releases and new features? We can do better than this! What about if you check all the feature names marked as new (`isNew`), and you compare them with an `QStringList` of seen-features stored in settings?